nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -47,6 +47,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
   The `nimPackages` and `nim2Packages` sets have been removed.
   See https://nixos.org/manual/nixpkgs/unstable#nim for more information.
 
+- Following options of the Nextcloud module are now covered by `extraOptions` and should be migrated: `logLevel` (now `extraOptions.loglevel`), `logType` (`extraOptions.log_type`), `defaultPhoneRegion` (`extraOptions.default_phone_region`), `overwriteProtocol` (`extraOptions.overwriteProtocol`), `skeletonDirectory` (`extraOptions.skeletondirectory`), `globalProfiles` (`extraOptions.profile.enabled`), `extraTrustedDomains` (`extraOptions.trusted_domains`) and `trustedProxies` (`èxtraOptions.trusted_proxies`).
```

Comment:
How about "the following options of the Nextcloud module were moved into [`services.nextcloud.extraOptions`](#opt-services.nextcloud.extraOptions) and match the upstream option names"?